admin/load.php

```diff
@@ -757,58 +791,74 @@ function perflab_plugin_admin_notices() {
 		<?php
 	}
 
-	$active_modules_with_plugins = perflab_get_active_modules_with_standalone_plugins();
+	$available_module_names = perflab_get_modules_compatible_with_standalone_plugin();
+	if ( empty( $available_module_names ) ) {
+		return;
+	}
 
-	if ( ! empty( $active_modules_with_plugins ) ) {
-		$module_data            = perflab_get_modules();
-		$available_module_names = array();
-		foreach ( $active_modules_with_plugins as $module_slug ) {
-			if ( isset( $module_data[ $module_slug ] ) && ! perflab_is_standalone_plugin_loaded( $module_slug ) ) {
-				$available_module_names[] = $module_data[ $module_slug ]['name'];
-			}
-		}
+	$modules_count = count( $available_module_names );
+	if ( $modules_count < 1 ) {
+		return;
+	}
```

Comment:
This is useless now: We already check `empty` above, and `count( ... ) < 1` will always result in the same for an array.

Since we need the module count below, we should probably just remove the `empty` check above.

Reply:
Thanks for the feedback.

I have removed the `$modules_count < 1` check as it is duplicate of empty check.